.travis.yml

```diff
@@ -8,30 +8,28 @@ cabal: "3.0"
 
 cache:
   directories:
-  - "$HOME/.cabal/store"
-  - "$HOME/.stack"
-  - "$TRAVIS_BUILD_DIR/.stack-work"
+    - "$HOME/.cabal/store"
+    - "$HOME/.stack"
+    - "$TRAVIS_BUILD_DIR/.stack-work"
 
 matrix:
   include:
+    # Cabal
+    - ghc: 8.2.2
+    - ghc: 8.4.4
+    - ghc: 8.6.5
+    - ghc: 8.8.1
 
-  # Cabal
-  - ghc: 8.2.2
-  - ghc: 8.4.4
-  - ghc: 8.6.5
-  - ghc: 8.8.1
-
-  # stack
-  - ghc: 8.8.1
-    env: STACK_YAML="$TRAVIS_BUILD_DIR/stack.yaml"
+    # stack
+    - ghc: 8.8.1
+      env: STACK_YAML="$TRAVIS_BUILD_DIR/stack.yaml"
 
 install:
   - |
     if [ -z "$STACK_YAML" ]; then
       ghc --version
       cabal --version
       cabal update
-      cabal build --enable-tests --enable-benchmarks
```

Comment:
Let's not remove this line :slightly_smiling_face: I understand that `cabal test --enable-test` will the project anyway, but it's better to split it into multiple stages for various reasons. However, you can remove `--enable-benchmarks` flag